tqdm/_tqdm.py

```diff
@@ -289,9 +289,20 @@ def format_meter(n, total, elapsed, ncols=None, prefix='',
 
             # format the stats displayed to the left and right sides of the bar
             l_bar = (prefix if prefix else '') + \
-                '{0:3.0f}%|'.format(percentage)
-            r_bar = '| {0}/{1} [{2}<{3}, {4}{5}]'.format(
-                    n_fmt, total_fmt, elapsed_str, remaining_str, rate_fmt,
+                '{0:6.2f}%|'.format(percentage)
```

Comment:
I wanted more detailed percentage number because I need it for big number of iterations